### PR TITLE
Task 4 - modals and forms

### PR DESCRIPTION
**Pull request by apyd**
Done:

- [x] - Styles and markup are implemented
- [x] - Used stateless/stateful approach
- [x] - Used event handlers
- [x] - Used React hooks

Not done: N/A

### CODE REVIEW

File: src/components/Modal/Modal.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import "./Modal.scss";
+import { ButtonType } from "../UI/Button/Button.consts";
+
+export const Modal: FC<IModalProps> = ({
+  title,
+  isOpened,
+  toggleModal,
+  children,
+}) => {
+  const portalDiv = document.getElementById("modal") as HTMLElement;
```
Comment (Vladlan) [inline src/components/Modal/Modal.tsx:14]: "modal" is not suitable for an id. Please find something more unique and  store it as const

File: src/components/Modal/Modal.types.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { ReactNode } from "react";
+
+export interface IModalProps {
+    title: string,
+    isOpened: any,  // TODO - cannot be any
```
Comment (Vladlan) [inline src/components/Modal/Modal.types.ts:5]: isOpened: boolean

File: src/components/UI/MultiSelect/MultiSelect.tsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  selected,
+  toggleOption,
+  label,
+}) => {
+  const [isExpanded, setIsExpanded] = useState<boolean>(false);
+
+  const toggleSelect = (
+    event: React.MouseEvent<HTMLDivElement, MouseEvent>
+  ) => {
+    const target = document.getElementById("multi-select");
```
Comment (Vladlan) [inline src/components/UI/MultiSelect/MultiSelect.tsx:17]: The same here for "multi-select"

**Review by Vladlan — APPROVED** [review]

File: src/components/Modal/Modal.tsx (hunk 1 above)
Comment (apyd) [reply src/components/Modal/Modal.tsx:14]: In latest changes changed it to 'modalRoot' to use the same convention as 'root' for app div

File: src/components/UI/MultiSelect/MultiSelect.tsx (hunk 3 above)
Comment (apyd) [reply src/components/UI/MultiSelect/MultiSelect.tsx:17]: Thanks, decided to remove this part as it didn't add any benefits into component functionality so in latest changes there is no id for multiselect, also no target check

**Pull request merged**